mongoose/mongoose.d.ts

```diff
@@ -423,7 +423,7 @@ declare module "mongoose" {
 
   export interface Document {
     id?: string;
-    _id: Types.ObjectId;
+    _id: any;
```

Comment:
I prefer not `any` type.
In this case, we may use `number | string | mongoose.Types.ObjectId` instead of `any`.